tools/cldr-apps/pom.xml

```diff
@@ -0,0 +1,309 @@
+  <version>1.0</version>
+</dependency> -->
+    
+    <!-- db -->
+    <dependency>
+      <groupId>org.apache.derby</groupId>
+      <artifactId>derby</artifactId>
+      <version>10.10.1.1</version>
+      <!-- <scope>test</scope> -->
+    </dependency>
+
+    <dependency>
+    <groupId>mysql</groupId>
+    <artifactId>mysql-connector-java</artifactId>
+    <version>8.0.21</version>
+    </dependency>
+
+<!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
+<dependency>
+    <groupId>com.google.guava</groupId>
+    <artifactId>guava</artifactId>
+    <version>28.1-jre</version>
+</dependency>
+<!-- Note: failureaccess comes in automatically. -->
+
+<!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
+<dependency>
+    <groupId>com.google.code.gson</groupId>
+    <artifactId>gson</artifactId>
+    <version>2.8.5</version>
+</dependency>
+
+<dependency>
+    <groupId>com.google.myanmartools</groupId>
+    <artifactId>myanmar-tools</artifactId>
+    <version>1.1.1</version>
+</dependency>
+
+<dependency>
+    <groupId>xerces</groupId>
+    <artifactId>xercesImpl</artifactId>
+    <version>2.12.0</version>
+</dependency>
+
+    
+    
+  </dependencies>
+
+  <build>
+    <finalName>cldr-apps</finalName>
+    <sourceDirectory>src</sourceDirectory> <!-- TODO: HACK! until we can pivot src dirs -->
+    <!-- No resources… they would end up under classes -->
+    <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
+      <plugins>
+        <plugin>
+          <artifactId>maven-clean-plugin</artifactId>
+          <version>3.1.0</version>
+        </plugin>
+        <!-- see http://maven.apache.org/ref/current/maven-core/default-bindings.html#Plugin_bindings_for_war_packaging -->
+        <plugin>
+          <artifactId>maven-resources-plugin</artifactId>
+          <version>3.0.2</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.8.0</version>
+          <configuration>
+            <includes>
+                <include>org/**/*.java</include>
+            </includes>
+          </configuration>
+        </plugin>
+        <plugin>
+          <artifactId>maven-surefire-plugin</artifactId>
+	  <version>${maven-surefire-plugin-version}</version>
+          <configuration>
+            <systemPropertyVariables>
+              <CLDR_DIR>${project.basedir}/../../</CLDR_DIR>
+              <CLDR_ENVIRONMENT>UNITTEST</CLDR_ENVIRONMENT>
+              <java.awt.headless>true</java.awt.headless>
+              <!-- <rununittest.arg>-n -q</rununittest.arg> -->
+            </systemPropertyVariables>
+          </configuration>
+        </plugin>
+        <plugin>
+          <artifactId>maven-war-plugin</artifactId>
+          <version>3.2.2</version>
+        <configuration>
+          <webResources>
+            <resource>
+              <!-- this is relative to the pom.xml directory -->
```

Comment:
"the pom.xml directory" --> "the directory this POM file is in" ?

Reply:
I ended up removing/replacing this comment. It's more important to note that the point of this is to temporarily avoid renaming all of the source files.